webroot/results/admin/upload_results.php

```diff
@@ -7,7 +7,7 @@
 adminHeadline('Upload Competition Results &amp; Scrambles');
 
 $scripts = new WCAClasses\WCAScripts();
-$scripts->add('data_upload_help.js');
+$scripts->add('data_upload_help.js?ver=201802082346');
```

Comment:
Won't this result in two url parameters (both `v=` and `ver=`?) Would the solution instead be to bump the version in [`webroot/results/includes/WCAClasses/WCAScripts.class.php`](https://github.com/jfly/worldcubeassociation.org/blob/285c4a745f20ffd5537b10e3a1f8d925c0877ae0/webroot/results/includes/WCAClasses/WCAScripts.class.php#L10)?

I'm surprised we even need this cache busting technique, I checked and it looks like our server isn't aggressively caching this stuff:

![image](https://user-images.githubusercontent.com/277474/36015448-4aa3c88e-0d23-11e8-85da-b1eaa84712f0.png)

But whatever, let's not waste too much time improving this ancient code =)